azure-pipelines.yml

```diff
@@ -142,6 +142,8 @@ stages:
         displayName: 'Use Python 3.9'
         inputs:
           versionSpec: '3.9'
+      - bash: sudo apt install -y pandoc graphviz
```

Comment:
I'm a bit 50/50 between adding a dependency and adding complexity to the build and test chain. Since `graphviz` seems OK as a dependency, I'd opt to just depend on it so that the code is always executed reliably when tests run.

Reply:
Is it easy enough to install on Windows - i.e. does the installer .exe work as advertised, and also together with the python wrapper library?

Reply:
I have it running on my Win 10 system and don't recall any issues with the installer. The Python wrapper is also just `pip install ...`, and then calls the binary.

Reply:
Having `.visualize()` is generally nice when working with Dask arrays, so it shouldn't be too much of a burden for LiberTEM devs. :-)

Reply:
Perfect, then I'm fine with depending on graphviz :+1: